app/src/main/java/com/kms/cura/model/request/UserModelResponse.java

```diff
@@ -0,0 +1,45 @@
+package com.kms.cura.model.request;
+
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.kms.cura.constant.EventConstant;
+import com.kms.cura.entity.user.PatientUserEntity;
+import com.kms.cura.entity.user.UserEntity;
+import com.kms.cura.event.EventBroker;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * Created by linhtnvo on 6/10/2016.
+ */
+public class UserModelResponse implements EntityModelResponse {
```

Comment:
Then change it name to LoginRegisterSomething as we will need more response for User